x-pack/plugins/infra/server/lib/sources/defaults.ts

```diff
@@ -17,4 +19,22 @@ export const defaultSourceConfiguration = {
     tiebreaker: '_doc',
     timestamp: '@timestamp',
   },
+  logColumns: [
+    {
+      timestampColumn: {
+        id: '5e7f964a-be8a-40d8-88d2-fbcfbdca0e2f',
```

Comment:
I know these are just defaults and ultimately the ID doesn't map to anything in terms of retrieval, even so I wonder if down the line this would become confusing? I.e. someone thinking these hardcoded IDs are special and map to something specific somewhere. Any harm in generating the default with the UUID library?

Reply:
Good question. I could imagine it would confuse the UI code if there were multiple Kibana instances behind a load balancer (which is a supported setup) and the api returned different column ids between requests even though the config hasn't changed.

Reply:
Gotcha - thanks for the explanation.